packages/debugger/package.json

```diff
@@ -19,9 +19,9 @@
     "docs": "esdoc",
     "prepare": "yarn build",
     "start": "node ./webpack/dev-server.js",
-    "test": "mocha-webpack --webpack-config webpack/webpack.config-test.js --recursive; STATUS=$?; rm -r dist; yarn build; (exit $STATUS)",
-    "test:coverage": "nyc mocha-webpack --webpack-config webpack/webpack.config-test.js --recursive",
-    "test:debug": "NODE_ENV=testing node --inspect ./node_modules/.bin/mocha-webpack --webpack-config webpack/webpack.config-test.js --recursive"
+    "test": "mochapack --webpack-config webpack/webpack.config-test.js --recursive && yarn build; (exit $STATUS)",
```

Comment:
So now that we're using mochapack, we should be able to get rid of the junk on this line, right?  We should be able to just do `mochapack --webpack-config webpack/webpack.config-test.js --recursive`?  The reason for all the rest was due to handling some problems caused by mocha-webpack, and the `(exit $STATUS)` doesn't even make sense anymore.

(OK, I just checked, the problem is indeed gone and we can get rid of all this.)

Reply:
I'll remove this.